modules/nw-gcp-installing-global-access-configuration.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-gcp-vpc.adoc
+// * installing/installing-restricted-networks-gcp
+
+[id="nw-gcp-global-access-configuration_{context}"]
+= Create an Ingress Controller with global access on GCP
+You can create an Ingress Controller with global access on a new GCP cluster.
+
+.Prerequisites
+
+* You created the `install-config.yaml` and complete any modifications to it.
+
+.Procedure
+
+Create an Ingress Controller with global access on a new GCP cluster.
+
+. Change to the directory that contains the installation program and create a manifest file:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory> <1>
+----
+<1> For `<installation_directory>`, specify the name of the directory that
+contains the `install-config.yaml` file for your cluster.
++
+After creating the file, several network configuration files are in the
+`manifests/` directory, as shown:
++
+[source,terminal]
+----
+$ ls <installation_directory>/manifests/cluster-ingress-default-ingresscontroller.yaml
+----
++
+.Example output
+[source,terminal]
+----
+cluster-ingress-default-ingresscontroller.yaml
+----
+
+. Open the `cluster-ingress-default-ingresscontroller.yaml` file in an editor and enter a custom resource (CR) that describes the Operator configuration you want:
++
+.Sample `clientAccess` configuration to `Global`
+[source,yaml]
+----
+  spec:
+    endpointPublishingStrategy:
+      loadBalancer:
+        providerParameters:
+          gcp:
+            clientAccess: Global <1>
+          type: GCP
+        scope: Internal
```

Comment:
Might want to take the opportunity to call out the fact that global access is only applicable to internal scope load balancers with a <2> callout here. Thoughts?

Reply:
+1

Reply:
@sgreene570 agree, i will add that callout. ty!! :)

Reply:
Fix looks good!